### DISCUSSION TITLE
Mina 黑客松火热进行中 🔥

### DISCUSSION THREAD

**Discussion by phoouze** [openbuildxyz/mina-zkapp-bootcamp#350] [Announcements]
无论如何先[报名](https://forms.gle/dY5JPZrSftqLpbLR7)，或许你能做出牛逼的项目呢！🚀 

没有想法？没有队友？ 不用担心，我们会组织大家一起讨论项目idea，帮助大家匹配到合适的队伍。

在这里请大胆分享你的创意想法，自己会的技术栈，甚至可以介绍你曾经参与过的项目，快速寻找适合你的队友！

idea:
1. 利用Mina链的zk-SNARKs，创建一个去中心化的身份验证系统，用户无需提供个人数据，只需证明身份即可
2. GameFi，所有游戏数据通过Mina链记录，使用zk-SNARK保证玩家数据隐私
3. 构建一个Mina链与其他链（如Ethereum）的跨链桥，允许用户在Mina链和其他链之间转移资产
4. 创建一个基于Mina链的匿名投票平台，确保投票数据的隐私性和不可篡改性
5. 内容发布平台，确保内容发布者的身份和内容本身的隐私不被泄露
6. Mina 交易签名工具， 帮助开发者生成并签名交易。特别适合开发者在没有完整钱包的情况下，签署和广播交易
7. Mina 数据可视化工具， 帮助用户更好地理解 Mina 区块链的运行状态、交易流动、矿工活动等数据
8. 代币生成工具，为用户提供一个简单的界面，能够在 Mina 上创建和管理代币
9. 监控工具， 实时监控 Mina 区块链的某些特定事件，如特定合约的调用、某个账户的交易、特定的区块生成等

一起头脑风暴，说出你的idea吧

**Comment by liaobinbin** [comment]
求大佬带，一起做个钓鱼网站，把这届学员赚到的币全部骗光

**Comment by super-sheng** [comment]
有没有觉得 做一个 学习计划 相关的项目， 比如 叫什么 百日学习计划， 然后可以在上面学习各种他感兴趣的内容， 可以打赏， 可以评价，然后可以发纪念卡 之类的？

**Comment by wrule** [comment]
本人前端开发者，熟悉react, tailwindcss, nextjs, remix, 等等。熟悉cloudflare，aws。平时使用viem，wagmi，rainbowkit，tailwindcss做项目，最近深入研究ai，可以做出不少动画特效。也写过一些Solidity的个人项目。有一定量化交易系统设计经验。如果有感兴趣的可以招募我为组员，一起合作推进项目。

**Reply by liaobinbin** [reply]
大佬是不一样，会这么多

**Comment by veithly** [comment]


**Reply by phoouze** [reply]
这个不错，可以在群里拉队友一起搞

**Reply by phoouze** [reply]
可以在群里喊喊，拉人一起来搞

**Reply by a2asister** [reply]
想参与进来啊

**Comment by kelh93** [comment]
本人前端开发者，熟悉react, tailwindcss, nextjs, nodejs等等。最近在学习Solidity，如果有感兴趣的可以招募我为组员，一起合作推进项目。

**Comment by a2asister** [comment]
做个 DeFi 或者 GameFi？算了太复杂了，还是做个投票系统吧？

**Reply by phoouze** [reply]
投票系统不就是task2吗
![image](https://github.com/user-attachments/assets/9e62685e-a783-44a8-90bd-14025ac1666d)
